uReplicator-Common/src/main/java/com/uber/stream/kafka/mirrormaker/common/core/KafkaBrokerTopicObserver.java

```diff
@@ -75,7 +78,19 @@ public KafkaBrokerTopicObserver(String brokerClusterName, String zkString) {
       public void run() {
         tryToRefreshCache();
       }
-    }, 0, 60, TimeUnit.SECONDS);
+    }, 0, refreshTimeIntervalInMillis, TimeUnit.MILLISECONDS);
+    registerMetric();
+  }
+
+  private void registerMetric() {
+    try {
+      KafkaUReplicatorMetricsReporter.get().registerMetric(
+              String.format(METRIC_TEMPLATE, _kakfaClusterName, "refreshLatency"), _refreshLatency);
+      KafkaUReplicatorMetricsReporter.get().registerMetric(
+              String.format(METRIC_TEMPLATE, _kakfaClusterName, "kafkaTopicsCounter"), _kafkaTopicsCounter);
+    } catch (Exception e) {
+      LOGGER.error("Failed to register metrics to HelixKafkaMirrorMakerMetricsReporter " + e);
```

Comment:
`HelixKafkaMirrorMakerMetricsReporter " + e` to `HelixKafkaMirrorMakerMetricsReporter ", e`